host/elastic_search_test.go

```diff
@@ -18,9 +18,6 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-//go:build esintegration
-// +build esintegration
```

Comment:
seems pretty likely an improvement (build flags are quite problematic imo).  :+1: assuming no surprises in CI or something.

Reply:
actually, I removed this for my local tests. Buildkite is just working fine, but it might be annoying to start those tests locally without ES running, I might revert this change

Reply:
yeah, what I'd prefer to do here is to always build everything (greatly improves IDE behavior, and ensures breaking refactors are actually complete), but skip tests that don't have the necessary env setup running.

but 👍 that's an unrelated change to this PR.